package/rtcm3_out_protocol/src/info_rtcm3_out.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2016 Swift Navigation Inc.
+ * Contact: Jacob McNamee <jacob@swiftnav.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+const char *protocol_name = "RTCM3Out";
+const char *setting_name = "RTCMv3 OUT";
+
+int port_adapter_opts_get(char *buf, size_t buf_size, const char *port_name)
+{
+  return snprintf(buf,
+                  buf_size,
+                  "-p 'ipc:///var/run/sockets/rtcm3_external.sub' "
```

Comment:
Nit: whitespace

Reply:
we can remove the `-p` option here entirely in order to make it unidirectional. will also need to remove this line in the other protocol package:
https://github.com/swift-nav/piksi_buildroot/blob/cc4b5088df281148ef8e74e30718abf787db9a5b/package/rtcm3_protocol/src/info_rtcm3.c#L28